src/organisms/Accordion/__tests__/accordion.spec.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { mount } from 'enzyme';
+
+// import sinon from 'sinon';
```

Comment:
why added?

Reply:
(Same as above)
Fixed this - accidentally added it back in when I rebased.